src/components-styled/sidebar-metric/sidebar-barscale.tsx

```diff
@@ -0,0 +1,54 @@
+import { get } from 'lodash';
+import { BarScale } from '~/components/barScale';
+import siteText from '~/locale/index';
+import { assert } from '~/utils/assert';
+import { Box } from '../base';
+import { BarScaleConfig } from './data-config';
+
+interface SidebarBarScaleProps {
+  config: BarScaleConfig;
+  value: number;
+  localeTextKey: string;
```

Comment:
```suggestion
  localeTextKey: keyof TALLLanguages;
```
That will save you the `assert(text, 'No text found for locale key ${localeTextKey}')` call